package-lock.json

```diff
@@ -76,13 +76,270 @@
       "dev": true,
       "license": "MIT"
     },
```

Comment:
Please remove the unrelated changes here.

Reply:
The file is still there, still with a massive diff.

Reply:
I apologize, but I did not add these files with PR.